add: react custom hook

narendraprasadb opened the pull request:
-add custom hook
![Screenshot 2024-07-25 163305](https://github.com/user-attachments/assets/8398833c-179b-419b-8d33-ecd601244ada)

Commit:
add: react custom hook

SaiCharanChetpelly31 reviewed (commented):
Make sure you follow git best practices for naming branches and creating commit messages.
Focus on naming props.
Try to place hardcoded strings in constants.

SaiCharanChetpelly31 commented on react-hook/my-react-app/CustomHook/SetHook/index.ts:
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { useState } from 'react';
+const useHook=(p0: number)=>{
```
GIve proper line space

SaiCharanChetpelly31 commented:
@narendraprasadb Dont forget to followup this PR.

Commit:
Refactor: Move hardcoded values to constants file and clean up code

- Moved hardcoded values into a separate constants file for better maintainability.
- Cleaned up code formatting for improved readability.
